test/wanda_web/controllers/v1/checks_customization_controller_test.exs

```diff
@@ -0,0 +1,278 @@
+defmodule WandaWeb.V1.ChecksCustomizationsControllerTest do
+  use WandaWeb.ConnCase, async: true
+
+  import OpenApiSpex.TestAssertions
+
+  alias WandaWeb.Schemas.V1.ApiSpec
+
+  setup do
+    %{api_spec: ApiSpec.spec()}
+  end
+
+  describe "checks customization" do
+    test "should not accept invalid body", %{conn: conn, api_spec: api_spec} do
+      check_id = "ABC123"
+      group_id = Faker.UUID.v4()
+
+      missing_values_filed_errors = [
+        %{
+          title: "Invalid value",
+          source: %{
+            pointer: "/values"
+          },
+          detail: "Missing field: values"
+        }
+      ]
+
+      invalid_body_scenarios = [
+        %{
+          body: nil,
+          expected_errors: missing_values_filed_errors
+        },
+        %{
+          body: %{},
+          expected_errors: missing_values_filed_errors
+        },
+        %{
+          body: "",
+          expected_errors: missing_values_filed_errors
+        },
+        %{
+          body: "{}",
+          expected_errors: missing_values_filed_errors
+        },
+        %{
+          body: %{
+            values: []
+          },
+          expected_errors: [
+            %{
+              title: "Invalid value",
+              source: %{
+                pointer: "/values"
+              },
+              detail: "Array length 0 is smaller than minItems: 1"
+            }
+          ]
+        },
+        %{
+          body: %{
+            values: nil
```

Comment:
What do you think about adding a test to check for the missing name ? 
Example:
```%{
  body: %{values: [%{value: 50}]}
  .....
  ```
  
  and 
  ```
  ```%{
  body: %{values: [%{name: "blabla"}]}
  .....
  ```
  ```

Reply:
Here we go a988f8d9a0d034f18bb392133f9ae2a5d679b4a3